_pages/clients/nostr.console.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Nostr Console
+permalink: /nostr.console/
+web:
+github: vishalxl/nostr_console
+instances:
+tags:
+- client
```

Comment:
So it is social but also ... chat and dm?
Many others support dm, too ...

```suggestion
- client
- social
- dm
- chat
```

Reply:
![image](https://user-images.githubusercontent.com/44260360/225302016-70c39cf1-d38a-4849-bd71-a260ff35c3b0.png)